resources/dicts/events/injury/mountainous.json

```diff
@@ -117,5 +117,40 @@
                 "reg_death": "m_c died from {PRONOUN/m_c/poss} wounds after falling from a tall cliff."
         }
         ]
-    }    
+    },
+    {
+        "event_id": "mtn_injury_war_cliffsaved",
+        "location": ["mountainous"],
+        "season": ["any"],
+        "sub_type": ["war"],
+        "weight": 20,
+        "event_text": "m_c acts without thinking, yanking the o_c_n warrior {PRONOUN/m_c/subject}'d been fighting back from the edge of the cliff and saving their life.",
```

Comment:
could do with a version of this event in the beach biome, featuring an ocean cliff and roiling waters below.